pkg/controller/elasticsearch/driver/downscale_invariants_test.go

```diff
@@ -227,7 +227,8 @@ func Test_checkDownscaleInvariants(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			canDownscale, reason := checkDownscaleInvariants(*tt.state, tt.statefulSet)
+			toDelete, reason := checkDownscaleInvariants(*tt.state, tt.statefulSet, 1)
+			canDownscale := toDelete > 0
```

Comment:
let's maybe test the exact value here to avoid any unpleasant surprise and make sure we test the contract